code/datums/mutations/cold_resistance.dm

```diff
@@ -1,11 +1,12 @@
 //Cold Resistance gives your entire body an orange halo, and makes you immune to the effects of vacuum and cold.
 /datum/mutation/human/cold_resistance
 	name = "Cold Resistance"
+	desc = "A strange mutation that makes the holder completely immune to hostile temperatures."
```

Comment:
it's probably a good moment to rename it to Space Adaptation or something like that, since it also prevents damage from low pressure